Reorganize InfluxDB Clustered installation documentation (#5636)
influxdata/docs-v2: content/influxdb/clustered/admin/licensing.md

weight: 101
influxdb/clustered/tags: [licensing]
related:
  - /influxdb/clustered/install/set-up-cluster/licensing/
  - /influxdb/clustered/admin/upgrade/
---

Install and manage your InfluxDB Clustered license to authorize the use of
the InfluxDB Clustered software.

- [Install your InfluxDB license](#install-your-influxdb-license)
- [Verify your license](#verify-your-license)
  - [Verify database components](#verify-database-components)

>   - [Verify the Secret exists ](#verify-the-secret-exists-)

Reviewer: ```suggestion
  - [Verify the Secret exists ](#verify-the-secret-exists)
```

{{% note %}}
If setting up an InfluxDB Clustered deployment for the first time, first
[set up the prerequisites](/influxdb/clustered/install/set-up-cluster/licensing/) and
[configure your cluster](/influxdb/clustered/install/set-up-cluster/configure-cluster/).

> After your InfluxDB namespace is created and prepared, you will be able to

Reviewer: ```suggestion
After your InfluxDB namespace is created and prepared, you can 
```

After you [install your license](#install-your-influxdb-license),
run the following command to check that database pods start up and are in the
`Running` state:

<!--pytest.mark.skip-->

```bash
kubectl get pods -l app=iox --namespace influxdb
```

If a `Pod` fails to start, run the following command to view pod information:

<!--pytest.mark.skip-->

{{% code-placeholders "POD_NAME" %}}

```sh
kubectl describe pod POD_NAME --namespace influxdb
```

{{% /code-placeholders %}}

### Verify the `Secret` exists 

Run the following command to verify that the licensing activation created a
`iox-license` secret:

<!--pytest.mark.skip-->

```sh
kubectl get secret iox-license --namespace influxdb
```

If the secret doesn't exist,
[view `license-controller` logs](#view-license-controller-logs) for more
information or errors.

### View `license controller` logs

The `license controller` component creates a `Secret` named `iox-license` from
your `License`. To view `license controller` logs for troubleshooting, run the
following command:

<!--pytest.mark.skip-->

```sh
kubectl logs deployment/license-controller --namespace influxdb
```

## Recover from a license misconfiguration

> If you deploy a licensed release of InfluxDB Clustered with an invalid or

Reviewer: ```suggestion
If you deploy a licensed release of {{% product-name %}} with an invalid or
```
